src/editor/editor.cpp

```diff
@@ -351,17 +337,44 @@ namespace Editor
             }
 
             ImGui::Image(target_texture, autotiler_window_size_previous_frame);
+
             screen_mouse_pos = ImGui::GetCursorScreenPos();
```

Comment:
Shoudln't this be the imgu window (Screen) pos? not the mouse pos?

Reply:
changed to imgui_window_global_offset. For further clarification see the reply above.

Reply:
did u push up that change? Still looks like `screen_mouse_pos`